src/Engine/ProtoCore/Properties/Resources.en-US.resx

```diff
@@ -117,9 +117,81 @@
   </resheader>
+  <data name="attributeArgMustBeConstant" xml:space="preserve">
+    <value>An attribute argument must be a constant expression</value>
+  </data>
+  <data name="baseIsExpectedToCallBaseConstructor" xml:space="preserve">
+    <value>'base' is expected here to call base constructor.</value>
+  </data>
+  <data name="CheckIfTheElementsInListAreSameType" xml:space="preserve">
+    <value>Checks if all the elements in the specified list have the same type.</value>
+  </data>
+  <data name="ChecksIfLengthsAreSameInMultiDimentionalList" xml:space="preserve">
+    <value>Checks if each of rows in multidimentional list has the same number of elements</value>
+  </data>
+  <data name="ChecksIfListContainsTheElement" xml:space="preserve">
+    <value>Checks if the list contains the given element</value>
+  </data>
+  <data name="ChecksIfTheListIsAllFalse" xml:space="preserve">
+    <value>Checks if all elements in the specified list are false</value>
+  </data>
+  <data name="ChecksIfTheListIsAllTrue" xml:space="preserve">
+    <value>Checks if all elements in the specified list are true</value>
+  </data>
+  <data name="ChecksListWithUniformDepth" xml:space="preserve">
+    <value>Checks if the list has a uniform depth</value>
+  </data>
+  <data name="closeBracketExpected" xml:space="preserve">
+    <value>')' expected - Imcomplete Closure</value>
+  </data>
+  <data name="ContainsKeys" xml:space="preserve">
+    <value>Checks if a list contains a specific key</value>
+  </data>
+  <data name="DeterminesObjectsAreEqual" xml:space="preserve">
+    <value>Determines whether two object instances are equal</value>
+  </data>
+  <data name="emptyCharacterLiteral" xml:space="preserve">
+    <value>Empty character literal.</value>
+  </data>
+  <data name="failedToImport" xml:space="preserve">
+    <value>Failed to import {0}</value>
+  </data>
+  <data name="functionCallCannotBeAtLeftSide" xml:space="preserve">
+    <value>function call is not allowed at the left hand side of an assignment</value>
+  </data>
   <data name="FUNCTION_GROUP_RESOLUTION_FAILURE" xml:space="preserve">
     <value>No function called {0} could be found. Please check the name of the function.</value>
   </data>
+  <data name="getKeys" xml:space="preserve">
+    <value>Gets all keys from an list</value>
+  </data>
+  <data name="gettypes" xml:space="preserve">
+    <value>Gets the Type of the current instance</value>
+  </data>
+  <data name="getValues" xml:space="preserve">
+    <value>Gets all values that stored in the list, including in dictionary</value>
+  </data>
+  <data name="ImportFileByGivenFilePath" xml:space="preserve">
+    <value>Imports from CSV by providing file path</value>
+  </data>
+  <data name="ImportFileByGivenFilePathWithMode" xml:space="preserve">
+    <value>Imports from CSV by providing file path. The result is transposed when trans is false</value>
```

Comment:
Imports data from a text file containing comma separated values into two-dimensional list and also transpose the output list if specified